builds/e2e/isa-95-smoke-test.yaml

```diff
@@ -57,9 +57,8 @@ stages:
     displayName: SettingUp level 5
     timeoutInMinutes: 180
     variables:
+      identityArtifactName: aziot-identity-ubuntu20.04-amd64
```

Comment:
Ubuntu 20.04 is EOL this April. We prob should move away from these sooner than later.

Reply:
Yes, agreed!